src/renderer/components/Tabs/index.tsx

```diff
@@ -0,0 +1,52 @@
+import './style.css';
+
+const Tabs = () => {
+  return (
+    <div className="Tabs">
+      <div className="tabs">
+        <div className="tab employeeTab">Employee</div>
+        <div className="tab subContractorTab">Sub Contractor</div>
+        <div className="tab projectTab">Project</div>
+        <div className="tab insuranceTab">Insurance</div>
+        <div className="tab positionTab">Position</div>
+      </div>
+
+      <div className="container">
+        <div className="containerStripe">&nbsp;</div>
+        <div className="containerTabs">
+          <div className="containerTab viewTab">View</div>
+          <div className="containerTab editTab">Edit</div>
+        </div>
+        <div className="containerContent">
+          <div className="containerFormRow">
+            <div className="formField">
+              <label htmlFor="field1">
+                Field1:&nbsp;
+                <input type="text" id="field1" />
```

Comment:
Shouldn't use IDs in a component, since you can theoretically render multiple at once

Reply:
Well really shouldn't use them at all, but yeah